mitodl/mitxpro
# allow marketing user to add/edit slug for resource pages

#### What are the relevant tickets?
Fixes #321 

#### What's this PR do?
Remove auto generation of slugs for resource pages.

#### How should this be manually tested?

- Add resource page in CMS.
- You should see promote_panel where you can add/edit slug of the page.

**Screenshot:**
<img width="1094" alt="Screen Shot 2019-05-23 at 11 58 09 AM" src="https://user-images.githubusercontent.com/4245618/58232067-fd7a8980-7d51-11e9-9869-eba1bfa342bd.png">
